File: docs/components_page/components/spinner/button.py
import dash_bootstrap_components as dbc
from dash import html

spinners = html.Div(
    [
        dbc.Button(
            dbc.Spinner(size="sm"),
            color="primary",
            disabled=True,
            className="me-1",
        ),
        dbc.Button(
            [dbc.Spinner(size="sm"), " Loading..."],
            color="primary",
            disabled=True,
        ),
    ]
)

File: docs/components_page/components/spinner/button.R
library(dashBootstrapComponents)
library(dashHtmlComponents)

spinners <- htmlDiv(
  list(
    dbcButton(
      dbcSpinner(size = "sm"),
      color = "primary",
      disabled = TRUE,
      className = "me-1"
    ),
    dbcButton(
      list(dbcSpinner(size = "sm"), " Loading..."),
      color = "primary",
      disabled = TRUE
    )
  )
)
